### ISSUE TITLE
This application was created by another user, not by Google

### ISSUE THREAD

**Issue by DavePaurillo** [enuchi/React-Google-Apps-Script#38]
![image](https://user-images.githubusercontent.com/61621124/81324067-01b9c700-90c9-11ea-9453-3d5cdf9fb74f.png)

I am deploying the project as a standalone webapp and I am getting this warning. I'm trying to figure out how to get rid of this but none work for me. Is this possible?

**Comment by enuchi** [comment]
Not sure. Can you share more info including what code you're running? Have not worked too much with webapps but they should work properly with this repo, as long as you have the `doGet`/`doPost` public functions and publish them properly.

See notes in #34 -- @amarjayr has been working with webapps and can maybe help debug.

**Comment by DavePaurillo** [comment]
I simply removed everything in code.gs and added this line the doGet function
`function doGet() {
  return HtmlService.createHtmlOutputFromFile("about")
}`
and I am able to show the about page on the webapp. I also tried to create a new webapp not with the boilerplate react and it also shows the warning. Anyway, I found [this](https://issuetracker.google.com/issues/63521070#comment4) and I think this is an intended behavior. Thank you!
